SoftLayer/CLI/vpn/ipsec/cancel.py

```diff
@@ -0,0 +1,30 @@
+"""Cancel a dedicated server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
+@click.option('--immediate',
+              is_flag=True,
+              default=False,
+              help="Cancels the server immediately (instead of on the billing anniversary)")
+@click.option('--reason',
+              help="An optional cancellation reason. See cancel-reasons for a list of available options")
+@click.option('--comment',
+              help="An optional comment to add to the cancellation ticket")
```

Comment:
I dont think we really need these options. If we need to send them into the API, just set them to something generic in the `manager.cancel_item` function.